packages/components/src/components/Pagination/Pagination.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import { Pagination } from "./Pagination";
+
+// eslint-disable-next-line lodash/prefer-noop, @typescript-eslint/no-empty-function
+const NOOP = (): void => {};
+
+describe("<Pagination />", () => {
```

Comment:
It would be good to also add some tests to make sure that on page change gets called with the right page number and clicking on a specific page, previous page, and next page buttons work.